internal/exec/terraform_outputs.go

```diff
@@ -216,60 +226,44 @@ func GetTerraformOutput(
 	skipCache bool,
 ) any {
 	stackSlug := fmt.Sprintf("%s-%s", stack, component)
-	message := fmt.Sprintf("Fetching %s output from %s in %s", output, component, stack)
 
 	// If the result for the component in the stack already exists in the cache, return it
 	if !skipCache {
 		cachedOutputs, found := terraformOutputsCache.Load(stackSlug)
 		if found && cachedOutputs != nil {
-			u.LogDebug(fmt.Sprintf("Found the result of the Atmos YAML function '!terraform.output %s %s %s' in the cache", component, stack, output))
+			l.Debug(fmt.Sprintf("Cache hit for '!terraform.output %s %s %s'", component, stack, output))
 			return getTerraformOutputVariable(atmosConfig, component, stack, cachedOutputs.(map[string]any), output)
 		}
 	}
 
-	// Initialize spinner
-	s := spinner.New()
-	s.Style = theme.Styles.Link
+	message := fmt.Sprintf("Fetching %s output from %s in %s", output, component, stack)
 
-	var opts []tea.ProgramOption
-	if !CheckTTYSupport() {
-		// set tea.WithInput(nil) workaround tea program not run on not TTY mod issue
-		opts = []tea.ProgramOption{tea.WithoutRenderer(), tea.WithInput(nil)}
-		u.LogTrace("No TTY detected. Falling back to basic output. This can happen when no terminal is attached or when commands are pipelined.")
-		fmt.Println(message)
+	if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
```

Comment:
This conditional does not belong here. It undoes the feature here: https://github.com/cloudposse/atmos/pull/947